Geometry/TrackerGeometryBuilder/src/TrackerGeomBuilderFromGeometricDet.cc

```diff
@@ -256,10 +256,10 @@ void TrackerGeomBuilderFromGeometricDet::buildGeomDet(TrackerGeometry* tracker)
         tracker->addDetId(composedDetId);
 
       } else if (gduTypeName.find("Lower") != std::string::npos) {
-        //FIXME::ERICA: the plane builder is built in the middle...
-        PlaneBuilderForGluedDet::ResultType plane = gluedplaneBuilder.plane(composed);
+        //The plane is *not* built in the middle, but on the Lower surface
+        Plane* plane = new Plane(dus->surface());
         composedDetId = theTopo->stack(gduId[i]);
-        StackGeomDet* stackDet = new StackGeomDet(&(*plane), dum, dus, composedDetId);
+        StackGeomDet* stackDet = new StackGeomDet(&(*plane), dus, dum, composedDetId);
```

Comment:
In the previous PR there was a note here from @cvuosalo , never answered: " Is there any point to the `&(*plane)` construction? Why not just write `plane,` which is a pointer? If there is a special side effect from this construction, it should be commented."

Reply:
`plane` is of type `GluedGeomDet::GluedGeomDet(PlaneBuilderForGluedDet::ResultType&` and can't therefore be passed directly as argument to this function. The `&(*plane) ` constructions gets around that.

Reply:
again. do not ask (or try to answer) this kind of questions here. Ask few lines above (code there since orca)